### DISCUSSION TITLE
Airdcpp windows no longer displays text, windows within the app don't seem to be working anymore

### DISCUSSION THREAD

**Discussion by AtariBaby** [airdcpp/airdcpp-windows#89] [General]
I no longer have visible text in Airdc++windows 4.20

I can no longer see the text on this app. Not sure why. If I click a window, I see no indication that the window has switched to another. I've tried uninstalling and reinstalling and it doesn't help.

<img width="726" alt="Screenshot 2023-06-20 115956" src="https://github.com/airdcpp/airdcpp-windows/assets/97095/6d6db45b-3983-457a-ba58-4a1cdefc1b53">

**Comment by maksis** [comment]
Try resizing the view by dragging the divider that is under the toolbar icons

**Comment by AtariBaby** [comment]
Oh, that was it! I wonder how that happens? Is there a universal hotkey that will expand them all if it happens again?Sent from my iPhoneOn Jun 20, 2023, at 12:44 PM, maksis ***@***.***> wrote:﻿
Try resizing the view by dragging the divider that is under the toolbar icons
